packages/peregrine/lib/talons/MiniCart/useMiniCart.js

```diff
@@ -25,15 +25,14 @@ export const useMiniCart = props => {
     const [{ cartId }] = useCartContext();
     const history = useHistory();
 
-    const {
-        data: miniCartData,
-        loading: miniCartLoading,
-        error: miniCartError
-    } = useQuery(miniCartQuery, {
-        fetchPolicy: 'cache-and-network',
-        variables: { cartId },
-        skip: !cartId
-    });
+    const { data: miniCartData, loading: miniCartLoading } = useQuery(
```

Comment:
I'm not sure if it was intentional that this query was popping a Toast on error, but it resulted in "The cart is inactive" being rendered occasionally, even though CartContext had already handled the recovery. I considered mapping the message to something more friendly, but since it wasn't guaranteed this would pop consistently, I decided to remove it entirely.